com.unity.template-hd/ProjectSettings/EditorBuildSettings.asset

```diff
@@ -6,6 +6,8 @@ EditorBuildSettings:
   serializedVersion: 2
   m_Scenes:
   - enabled: 1
-    path: Assets/SampleSceneAssets/Scenes/SampleScene.unity
+    path: Assets/Scenes/SampleScene.unity
```

Comment:
This line change is probably unnecessary AFAICT - can you revert that one line?

Reply:
Sure, can do - the (updated) path is correct though, the old one doesn't exist, is that fine?

Reply:
Ok if that's a correction then great lets land it as is. Thank you.